docs/snippets/react/my-component-story-configure-viewports.js.mdx

```diff
@@ -5,23 +5,27 @@ import React from 'react';
 
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
 
+import MyComponent from './MyComponent';
+
 export default {
-  title: 'Stories',
+  component: MyComponent,
   parameters: {
     //👇 The viewports object from the Essentials addon
     viewport: {
       //👇 The viewports you want to use
       viewports: INITIAL_VIEWPORTS,
       //👇 Your own default viewport
-      defaultViewport: 'iphone6'
+      defaultViewport: 'iphone6',
     },
   },
 };
 
-export const MyStory = () => <div />;
-MyStory.parameters = {
-  viewport: {
-    defaultViewport: 'iphonex'
+export const MyStory = {
+  parameters: {
+    viewport: {
+      defaultViewport: 'iphonex',
+    },
   },
+  render: () => <div />,
```

Comment:
```suggestion

```

Reply:
I'm good with it. It simplifies the story. Thank you for this 🙂